builtin/providers/digitalocean/resource_digitalocean_floating_ip.go

```diff
@@ -73,6 +73,40 @@ func resourceDigitalOceanFloatingIpCreate(d *schema.ResourceData, meta interface
 	return resourceDigitalOceanFloatingIpRead(d, meta)
 }
 
+func resourceDigitalOceanFloatingIpUpdate(d *schema.ResourceData, meta interface{}) error {
+	client := meta.(*godo.Client)
+
+	if v, ok := d.GetOk("droplet_id"); ok {
```

Comment:
We should probably check here whether there is a change prior to reassigning the floating IP?